src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
+        for (int i = 0; i < 3; i++) {
+            System.out.println(new Lottery().getRandomBall());
```

Comment:
create one instance of `Lottery`
```suggestion
            System.out.println(lottery.getRandomBall());
```